src/NexusMods.App.UI/WorkspaceSystem/Panel/PanelView.axaml.cs

```diff
@@ -32,6 +34,19 @@ public PanelView()
                 .Subscribe()
                 .DisposeWith(disposables);
 
+            // panel selection
+            this.AddDisposableHandler(PointerEnteredEvent, (_, _) =>
+            {
+                if (ViewModel is not null) ViewModel.IsSelected = true;
+            }, routes: RoutingStrategies.Direct | RoutingStrategies.Bubble, handledEventsToo: true).DisposeWith(disposables);
+
+            this.WhenAnyValue(view => view.IsKeyboardFocusWithin)
+                .Where(isFocused => isFocused)
+                .SubscribeWithErrorLogging(_ =>
+                {
+                    if (ViewModel is not null) ViewModel.IsSelected = true;
+                }).DisposeWith(disposables);
```

Comment:
This does not seem to be doing anything from my testing

Reply:
I tried using tab after disabling the pointer over version. I will try again if it is working on your end

Reply:
Ah, I found the problem. If you close the selected tab or open a new Workspace without any selection, it tabbing will not select the panels. If one panel is already selected, tabbing does select panels correctly.

Reply:
Actually pointer press doesn't seem to work either for new workspaces or if you close the selected panel

Reply:
> Actually pointer press doesn't seem to work either for new workspaces or if you close the selected panel

Can you write down in detail how to repro? If you close a selected panel, then the workspace should select the first panel you have.

Reply:
Closing selected panel, switching workspaces, and everything else works on my end.

Reply:
I tried cleaning my datamodel folder but now I can't get any panel to be selected after app start. No panel is selected and nothing selects panels

Reply:
Ah, I got it. If you have a single panel there is no selection highlighting, which was confusing me.
Is that by design?

Reply:
Yeah